examples/multi-datacenter/submit_replicator_dc1_to_dc2.sh

```diff
@@ -3,16 +3,18 @@
 HEADER="Content-Type: application/json"
 DATA=$( cat << EOF
 {
-  "name": "replicator-dc1-to-dc2",
+  "name": "replicator-dc1-to-dc2-topic1",
   "config": {
     "connector.class": "io.confluent.connect.replicator.ReplicatorSourceConnector",
     "topic.whitelist": "topic1,_schemas",
     "key.converter": "io.confluent.connect.replicator.util.ByteArrayConverter",
     "value.converter": "io.confluent.connect.replicator.util.ByteArrayConverter",
```

Comment:
key.converter, value.converter and header.converter are all worker level config. They should be set in docker-compose.yml, not in the connector configuration.